awscli/clidocs.py

```diff
@@ -159,6 +161,14 @@ def doc_options_start(self, help_command, **kwargs):
     def doc_option(self, arg_name, help_command, **kwargs):
         doc = help_command.doc
         argument = help_command.arg_table[arg_name]
+        if is_streaming_blob_type(argument.argument_model):
```

Comment:
Let's move this logic down to after the line `doc.include_doc_string(argument.documentation)`. Mainly when the documentation gets generated, the note shows up above the argument's documentation and it can be interpreted as a note for the previous argument, especially since the note does not explicitly use the argument name.